server/events/command/vcs.go

```diff
@@ -22,13 +22,14 @@ func (d *VCSStatusUpdater) UpdateCombined(ctx context.Context, repo models.Repo,
 	descrip := fmt.Sprintf("%s %s", strings.Title(cmdName.String()), d.statusDescription(status))
 
 	request := types.UpdateStatusRequest{
-		Repo:        repo,
-		PullNum:     pull.Num,
-		Ref:         pull.HeadCommit,
-		StatusName:  src,
-		State:       status,
-		Description: descrip,
-		DetailsURL:  "",
+		Repo:           repo,
+		PullNum:        pull.Num,
+		Ref:            pull.HeadCommit,
+		StatusName:     src,
+		State:          status,
+		Description:    descrip,
+		DetailsURL:     "",
+		PrCreationTime: pull.CreatedAt,
```

Comment:
Keep this consistent with `PullNum`. so `PullCreationTime`.

Ideally, we'd group these under one pull object though, but fine with leaving as is.

Reply:
Same for all others btw.

Reply:
Done